### ISSUE TITLE
Integration with other email providers, especially self-hosted

### ISSUE THREAD

**Issue by lukas-mertens** [twentyhq/twenty#5463]
## Scope & Context

**Example:** 
```
The ticket is about the gmail integration/email integration in general
```

## Current behavior

Currently it is only possible to integrate with gmail as it seems

## Expected behavior
I am part of an organization that runs most if its infrastructure self-hosted, including its email server. Twenty would be a great addition as a CRM. Currently it is not possible to integrate with self hosted email solutions however. An integration with mailcow/roundcube or simply imap (to allow integration with most email providers) would be great, so that not only gmail is supported.

**Comment by ajojotank** [comment]
I agree with this, although I am not sure on how to implement it as I am not a developer.

**Comment by FelixMalfait** [comment]
Moving to discussion. This is a highly requested feature. We need to continue working on the basics before we extend to other email providers but it will come at some point for sure!
